src/Components/SmartComponents/CVEs/CVEs.js

```diff
@@ -85,6 +85,10 @@ export const CVEs = ({ rbac }) => {
     };
 
     useEffect(() => {
+        // this line removes the specPath parameter which is injected by Cypress during testing
+        // it should have no effect on the actual application outside of Cypress tests
+        delete urlParameters.specPath;
```

Comment:
Could we try something like mentioned here and have test environment specific things not in the app code itself? https://github.com/cypress-io/cypress/issues/28021#issuecomment-1756646215

Reply:
I think the cypress index html[0] might be a better place to add something like this. 


[0] https://github.com/RedHatInsights/vulnerability-ui/blob/master/cypress/support/component-index.html

Reply:
Was looking for some solution like this, thank you!

Reply:
I put the line inside component test setup file, since we were already modifying the `window` object there.